cmd/subnetcmd/publish.go

```diff
@@ -451,3 +457,38 @@ func getVMInfo(sc *models.Sidecar) (*types.VM, error) {
 
 	return vm, nil
 }
+
+func getInfoForKnownVMs(strVer, repoName, vmBinDir, vmBin string) ([]string, *version.Semantic, string, string, error) {
+	maintrs := []string{"ava-labs"}
+	binPath := filepath.Join(vmBinDir, repoName+"-"+strVer, vmBin)
+	sha, err := getSHA256FromDisk(binPath)
+	if err != nil {
+		return nil, nil, "", "", err
+	}
+	ver, err := version.Parse(strVer)
+	if err != nil {
+		return nil, nil, "", "", err
+	}
+	dl := binutils.NewSubnetEVMDownloader()
```

Comment:
does this work for spaces?

Reply:
Good catch! It works (no error) but it stores the wrong URL